jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/CombinedResource.java

```diff
@@ -30,32 +27,34 @@
 import org.eclipse.jetty.util.URIUtil;
 
 /**
- * A collection of Resources.
- * Allows webapps to have multiple sources.
- * The first resource in the collection is the main resource.
- * If a resource is not found in the main resource, it looks it up in
- * the order the provided in the constructors
+ * Multiple resource directories presented as a single Resource.
  */
-public class ResourceCollection extends Resource
+public class CombinedResource extends Resource
 {
-    private final List<Resource> _resources;
-
     /**
-     * Instantiates a new resource collection.
-     *
-     * @param resources the resources to be added to collection
+     * <p>Make a Resource containing a combination of other resources</p>
+     * @param resources multiple resources to combine as a single resource. Typically, they are directories.
+     * @return A Resource of multiple resources or a single resource if only 1 is passed, or null if none are passed
+     * @see CombinedResource
      */
-    ResourceCollection(List<Resource> resources)
+    static Resource combine(List<Resource> resources)
```

Comment:
Should this also have a `Resource combine(Resource... resources)` same as the `ResourceFactory`?

Reply:
It's not a public API.  It is just there to move the implementation into the correct file.   So if the ... is not needed, no need to add it.